content/en/blog/_posts/2022-11-XX-runtime-observability/index.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: blog
+title: "Boosting Kubernetes container runtime observability with OpenTelemetry"
+date: 2022-11-XX
+slug: runtime-observability
+---
+
+**Authors:** Sascha Grunert
+
+When speaking about observability in the cloud native space, then probably
+everyone will mention [OpenTelemetry (OTEL)][otel] at some point in the
+conversation. That's great, because the community needs standards to rely on
+for developing all cluster components into the same direction. OpenTelemetry
+enables us to combine logs, metrics, traces and other contextual information
+(called baggage) into a single resource. Cluster administrators or software
+engineers can use this resource to get a viewport about what is going on in the
+cluster over a defined period of time. But how can Kubernetes itself make use of
+this technology stack?
+
+[otel]: https://opentelemetry.io
+
+Kubernetes consists of multiple components where some are independent and others
+are stacked together. Looking at the architecture from a container runtime
+perspective, then we see from the top to the bottom:
+
+- **kube-apiserver**: Validates and configures data for the API objects
+- **kubelet**: Agent running on each node
+- **CRI runtime**: Container Runtime Interface (CRI) compatible container runtime
+  like [CRI-O][crio] or [containerd][containerd]
+- **OCI runtime**: Lower level [Open Container Initiative (OCI)][oci] runtime
+  like [runc][runc] or [crun][crun]
+- **Linux kernel**: Underlying operating system
```

Comment:
Windows is an option here. Might want to mention that.

Reply:
Yep, added.